### PR TITLE
Add Drug Nodes 💊

### PR DESCRIPTION
**Pull request by laulopezreal**

### CODE REVIEW

**Comment by slobentanzer** [comment]
Hi @laulopezreal,
thanks for the addition! I'll have a look soon and merge accordingly.

**Comment by laulopezreal** [comment]
> Hi @laulopezreal, thanks for the addition! I'll have a look soon and merge accordingly.

Hi @slobentanzer,

Sorry for the lack of context in my PR. I am participating in a Hackathon, and my goal is to create a Knowledge Graph using the Open Targets Dataset. I came across BioCypher a while ago, so I decided to use it as the framework to build the OT KG.

I will be adding code to my fork to add more nodes and relations, along with documentation and data models, and some notebooks to run some experiments related to class imbalance in KG embedding models.

Is that something that could you guys might be interested to have in this repository? (Feel free to drop me a DM if you prefer to discuss it in private).

Thanks in advance for offering to have a look at the PR :)

**Comment by slobentanzer** [comment]
Hi @laulopezreal,
that sounds great. No worries about the PR, I was able to tell from your title and expressive commit messages. ;)

As this repo, at the moment, is very much a prototype, we are happy to include additional information. Notebooks and such should be in their own directory, but I don't see a reason against adding them, if they are broadly interesting for Open Targets users and drug discovery researchers. This is very much aligned with our general vision of BioCypher, so don't hesitate to open issues and PRs at any time.

You are also welcome to join our [Zulip](https://biocypher.zulipchat.com) chat for informal communication (if you have not done so already).

File: otar_biocypher/target_disease_evidence_adapter.py [hunk 1]
```diff
@@ -475,6 +506,9 @@ def get_nodes(self):
         # Diseases
         yield from self._yield_node_type(self.disease_df, DiseaseNodeField)
 
+        # Drugs
+        yield from self._yield_node_type(self.drug_df, DrugNodeField)
```
[the remark this answers is not in the document]
Comment (laulopezreal) [reply otar_biocypher/target_disease_evidence_adapter.py]: Got it 👍🏼  Thanks for your answer. My last commit solves the issue :)

**Comment by laulopezreal** [comment]
> Hi @laulopezreal, so far we are not writing any additional nodes because the setup is not complete. I made a specific comment at the location that needs to be updated.

Hi @slobentanzer, thanks a lot for the feedback. My last commit should fix it (I've tested it on my end and it worked) by adding the ontology class `chembl`

**Review by slobentanzer — APPROVED** [review]
@laulopezreal works for me as well, good job. merging and bumping the minor version. many thanks!

**Pull request merged**